### DISCUSSION TITLE
firewall  and Home network access

### DISCUSSION THREAD

**Discussion by allgrinder** [qdm12/gluetun#1133] [Q&A]
hey,
how can I set the firewall so that the containers can access an IP (192.168.178.95) in the home network. I'm afraid to set the firewall incorrectly.

Thanks and best regards

**Comment by qdm12** [comment]
Hey there, try with FIREWALL_OUTBOUND_SUBNETS=192.168.178.95/32

If it doesn't work, you might have to fiddle with `ip route` by execing in gluetun: `docker exec -it gluetun /bin/sh`
